hw/ip/rstmgr/dv/cov/rstmgr_cov_unr_excl.el

```diff
@@ -0,0 +1,437 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//
+// Generated UNR file from Synopsys UNR tool with coverage for rstmgr_cnsty_chk
+// excluded.
+//==================================================
+// This file contains the Excluded objects
+// Generated By User: maturana
+// Format Version: 2
+// Date: Fri Feb  4 09:26:11 2022
+// ExclMode: default
+//==================================================
+CHECKSUM: "1284271525 4124731032"
+INSTANCE: tb.dut
+ANNOTATION: "VC_COV_UNR"
+Toggle 0to1 tl_o.d_user.rsp_intg [6] "logic tl_o.d_user.rsp_intg[6:0]"
```

Comment:
these tl_o* show up because of missing `common_cov_excl`. we should add it back. please see my comment in rstmgr_sim_cfg.hjson

Reply:
Done, I'll regenerate the unr exclusion file.